backend/TreeOfAKind.Application/Command/Document.cs

```diff
@@ -6,11 +6,13 @@ public class Document
     {
         public Stream Content { get; }
         public string ContentType { get; }
+        public string Name { get; }
 
-        public Document(Stream content, string contentType)
+        public Document(Stream content, string contentType, string name)
         {
             this.Content = content;
             ContentType = contentType;
+            Name = name;
```

Comment:
Yeah, you have that here, but not in the API

Reply:
Name is taken from file